src/fidesops/ops/api/v1/endpoints/oauth_endpoints.py

```diff
@@ -214,9 +214,10 @@ def oauth_callback(code: str, state: str, db: Session = Depends(get_db)) -> None
         authentication = (
             connection_config.get_saas_config().client_config.authentication  # type: ignore
         )
-        auth_strategy: OAuth2AuthenticationStrategy = get_strategy(  # type: ignore
+        auth_strategy: OAuth2AuthorizationCodeAuthenticationStrategy = get_strategy(  # type: ignore
             authentication.strategy, authentication.configuration  # type: ignore
         )
-        auth_strategy.get_access_token(db, code, connection_config)
+        connection_config.secrets = {**connection_config.secrets, "code": code}  # type: ignore
```

Comment:
Adding the code outside of the `get_access_token` to make the function reusable between the different OAuth2 flows.